src/sentry/integrations/metric_alerts.py

```diff
@@ -86,7 +84,9 @@ def incident_attachment_info(incident, new_status: IncidentStatus, metric_value=
     if metric_value is None:
         metric_value = get_metric_count_from_incident(incident)
 
-    text = get_incident_status_text(alert_rule, metric_value)
+    text = ""
+    if status != INCIDENT_STATUS[IncidentStatus.CLOSED]:
+        text = get_incident_status_text(alert_rule, metric_value)
```

Comment:
From the ticket, it looks like we want to remove the status text only if it's not for a specific incident? So should we keep the text in place here?

Reply:
i think we don't want closed incidents either since we say "50 events in the last 5 minutes" but that count is based on the last incident which might have been closed a long time ago.

Reply:
<img width="656" alt="Screen Shot 2022-05-23 at 4 22 32 PM" src="https://user-images.githubusercontent.com/6288560/169919809-ba695ce2-cf72-4382-ab96-7639fa8182c2.png">
I guess my concern is that it seems like it's useful to show the condition for an alert when it fires/resolves. For unfurling an older alert I can see the concern, but I think the place this will be seen the most is as part of automated alerts.

Reply:
would make sense to only do this change in `metric_alert_attachment_info` since its only used for link unfurling

Reply:
Sounds good to me

Reply:
👍